trieChecker/README.md

```diff
@@ -0,0 +1,7 @@
+# How to use
```

Comment:
add also a description of what this tool does, before 'How to use'

Reply:
done